src/librustc_typeck/check/mod.rs

```diff
@@ -4075,7 +4079,7 @@ impl<'a, 'gcx, 'tcx> FnCtxt<'a, 'gcx, 'tcx> {
                     _ => Def::Err,
                 };
                 if item_name != keywords::Invalid.name() {
-                    self.report_method_error(span, ty, item_name, None, error);
+                    self.report_method_error(span, ty, item_name, None, error, None);
```

Comment:
Why are there no args here? Is it just that we don't have access to them, or are they inappropriate?